flpproto.sh

```diff
@@ -43,7 +44,8 @@ cmake $SOURCEDIR                                              \
     -DFairRoot_DIR=$FAIRROOT_ROOT                               \
     -DROOTSYS=$ROOTSYS \
     ${Configuration_ROOT:+-DConfiguration_ROOT=$Configuration_ROOT} \
-    ${Monitoring_ROOT:+-DMonitoring_ROOT=$Monitoring_ROOT}
+    ${Monitoring_ROOT:+-DMonitoring_ROOT=$Monitoring_ROOT} \
```

Comment:
You are absolutely right, yes. Of course the way you call your CMake variable is up to you, mine was a typo.